composer.json

```diff
@@ -30,7 +30,7 @@
     "extra": {
     },
     "require": {
-        "php": "~8.0.0 || ~8.1.0 || ~8.2.0"
+        "php": "~8.0.0 || ~8.1.0 || ~8.2.0 || ~8.3.0"
```

Comment:
@Ocramius 
Do we really still need version 8 here?

Reply:
No, but that's for another patch/day 😁

Reply:
I always go with "unless there is no need to drop it, why drop it".
If it prevents us from doing stuff, feel free to drop it but I don't like this "lets drop it because we can drop it" out of no reason.
But might be personal preference tho.

Reply:
We're dropping 8.0 slowly but steadily, it's just a different changeset / can be done in a separate release :)